warp/tests/geometry/test_hash_grid.py

```
@@ -475,6 +604,15 @@ def test_hashgrid_new_del(self):
 add_function_test(TestHashGrid, "test_hashgrid_dtype_validation", test_hashgrid_dtype_validation, devices=devices)
 add_function_test(TestHashGrid, "test_hashgrid_edge_cases", test_hashgrid_edge_cases, devices=devices)
 add_function_test(TestHashGrid, "test_hashgrid_negative_wrapping", test_hashgrid_negative_wrapping, devices=devices)
+add_function_test(
```

Ruff (0.15.5)

[error] `add_function_test` may be undefined, or defined from star imports

(F405)